samples/asgardeo-react-app/src/pages/home.tsx

```diff
@@ -45,7 +45,9 @@ export const HomePage: FunctionComponent<HomePagePropsInterface> = (
         signOut,
         getBasicUserInfo,
         getIDToken,
-        getDecodedIDToken
+        getDecodedIDToken,
+        on,
+        trySignInSilently,
```

Comment:
Any special reason to add `,` at the end?

Reply:
Trailing comma has been added by the IDE's formatter (and I'm used to add it too). 

> Trailing commas (sometimes called "final commas") can be useful when adding new elements, parameters, or properties to JavaScript code. If you want to add a new property, you can add a new line without modifying the previously last line if that line already uses a trailing comma. This makes version-control diffs cleaner and editing code might be less troublesome.


References:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas
- https://blog.logrocket.com/best-practices-using-trailing-commas-javascript/

Reply:
There is a lint rule to warn when there is are unnecessary commas ([comma-dangle rule](https://github.com/asgardeo/asgardeo-auth-react-sdk/blob/main/lib/.eslintrc.js#L55)). Check if your editor has properly configured the ESLint plugins.

```suggestion
        trySignInSilently
```

Reply:
Resolved by https://github.com/asgardeo/asgardeo-auth-react-sdk/pull/100